.github/workflows/semver_release.yml

```
@@ -0,0 +1,128 @@
+name: Make new release
+
+on:
+  push:
+    branches: [ "main" ]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  semver_labels:
+    name: Read semver labels from merged PR
+    runs-on: ubuntu-latest
+    environment: prod
+    outputs:
+      semver_app: ${{ steps.semver_labels.outputs.semver_app }}
+      semver_chart: ${{ steps.semver_labels.outputs.semver_chart }}
+      skip_release: ${{ steps.semver_labels.outputs.skip_release }}
+    steps:
+      - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744
+        with:
+          token: ${{ secrets.BOT_GITHUB_TOKEN }}
+      - uses: ./.github/actions/check-pr-semver-labels
+        id: semver_labels
+        with:
+          github_token: ${{ secrets.BOT_GITHUB_TOKEN}}
+  semver_update:
+    name: Update semver for every eligible file of repo (helm charts, gradle files, etc.)
+    runs-on: ubuntu-latest
+    needs: semver_labels
+    environment: prod
```

```suggestion
    if: ${{needs.semver_labels.outputs.skip_release == 'False'}}
    environment: prod
```

this step have to be skipped if skip release is set to true since this step perform a commit on main branch

---

Done in [59dd669](https://github.com/pagopa/pagopa-wallet-service/pull/215/commits/59dd6691d6d4d0a2b32779773f2b9a95a948caac)